### ISSUE TITLE
FEATURE REQUESTS: Please put all feature requests here!

### ISSUE THREAD

**Issue by finchMFG** [finchMFG/dewritoHub#3]
Cheers!

**Comment by jaron780** [comment]
Make the Page changer at the bottom show like 5 numbers at a time to skip between pages, and not show any more numbers if the next pages are empty (prevent going to page thats empty)

**Comment by finchMFG** [comment]
Great idea! Working on it!

**Comment by Phantom329** [comment]
Add a ranking system thats either upvote/downvote or out of 5 stars or something

**Comment by finchMFG** [comment]
There is currently a ranking system when you are logged in!

**Comment by Stuyk** [comment]
Throwing a recommendation out there for more diversification on categories.

Griffball, SWAT, etc.

Maybe subcategories for infection. Traditional, Minigame, etc.

**Comment by finchMFG** [comment]
I've gotten that suggestion a few times and have started work on it! Cheers!

**Comment by Stuyk** [comment]
Oh and I almost forgot another feature request.

D.Tube support for decentralized videos. Possibly featuring streamers from D.Live on the site too. :)

**Comment by finchMFG** [comment]
I like it! I've thought about extending the site to feature more functionality like hosting Tournaments, showing streams etc but never really got around to it. 

Thanks for the suggestion! I'll look into making the site an actual 'hub'

**Comment by iTzKomaBlader** [comment]
Could there be a chance to add a way to change the name of maps or a version number area to keep up with updates? Kinda like...

[Map Name] [Verson No. (Opt)]

**Comment by finchMFG** [comment]
Yup! I'm actually adding something like that in tonight's update! I'll stick on the version number too, since I hadn' thought of that. Cheers!

**Comment by lshort2** [comment]
Does anyone know why images are not loading for some maps (i.e. [Icebox Race](https://dewritohub.com/map/k3Zo9QXgT2uGxzxNT/icebox-race/))?

**Comment by finchMFG** [comment]
Those are old Archive maps. My program didn't upload them properly.
I'm going to strip them all of the site and re-upload them to fix the problems! Sorry about that!
